apps/keyboard/js/layouts/emoji.js

```diff
@@ -403,12 +475,104 @@ Keyboards.emoji = {
       ]
     ]
     }, {     // Page 4 - Symbols
-    panelKeys: [ { compositeKey: '🆗', type: 'emoji' }, { compositeKey: '🔀', type: 'emoji' },
-                 { compositeKey: '🔁', type: 'emoji' }, { compositeKey: '🔂', type: 'emoji' },
-                 { compositeKey: '🆕', type: 'emoji' }, { compositeKey: '🆙', type: 'emoji' },
-                 { compositeKey: '🆒', type: 'emoji' }, { compositeKey: '🆓', type: 'emoji' },
-                 { compositeKey: '🆖', type: 'emoji' }, { compositeKey: '📶', type: 'emoji' },
-                 { compositeKey: '🎦', type: 'emoji' }, { compositeKey: '🈁', type: 'emoji' }
+    panelKeys: [{ compositeKey: '3', type: 'emoji' }, { compositeKey: '0', type: 'emoji' },
```

Comment:
I can also see a Emoji glyph a number character follow by a VS16 on Apple Color Emoji, if you want to support it you could do that too (but VS16 has to included in the key definition, like other keys).